auth-spa/map.js

```diff
@@ -0,0 +1,45 @@
+
+import maplibregl from 'maplibre-gl';
+import { Deck } from '@deck.gl/core';
+import { CartoLayer, MAP_TYPES, setDefaultCredentials } from '@deck.gl/carto';
+
+export function createMap(accessToken) {
+  const INITIAL_VIEW_STATE = {
+    latitude: 39.8097343,
+    longitude: -98.5556199,
+    zoom: 4,
+    bearing: 0,
+    pitch: 30
+  };
+
+  setDefaultCredentials({
+    apiBaseUrl: 'https://gcp-us-east1.api.carto.com',
```

Comment:
A lot of people struggle with this url, if they create an account in another region.
I think it would be a good idea to create a const, with something like:

```suggestion
  const ACCOUNT_REGION_API_BASE = 'https://gcp-us-east1.api.carto.com'
  setDefaultCredentials({
    apiBaseUrl: ACCOUNT_REGION_API_BASE,
```